extensions/typescript-language-features/src/languageFeatures/organizeImports.ts

```diff
@@ -114,8 +175,8 @@ class ImportsCodeActionProvider implements vscode.CodeActionProvider {
 
 		this.fileConfigManager.ensureConfigurationForDocument(document, token);
 
-		const action = new vscode.CodeAction(this.title, this.kind);
-		action.command = { title: '', command: OrganizeImportsCommand.Id, arguments: [file, this.sortOnly] };
+		const action = new vscode.CodeAction(this.Command.title, this.kind);
```

Comment:
Where is this title exposed, if anywhere? It’s not the one that appears in the command palette.

Reply:
This title will show if you run `Source Action...` from the context menu or command palette